mustbe/verifier/index.js

```diff
@@ -72,6 +72,11 @@ Verifier.prototype.isAuthorized = function(activity, requestParams, cb){
 
     var validator = validators[activity];
     if (!validator){
+      // global "deny" or "allow" defined but not passed
+      if(denyAllowConfig.denier || denyAllowConfig.allower) {
+        return cb(null, false);
+      }
+
```

Comment:
To follow the original design, we have to define validators for all activities, except that the global "deny" or "allow" always returns true. But how about plenty of activities which have the same logic? 
For example, we save all the activities in database and assign roles for special users. We can define a global "allow" to check the user role and activity from database. For others who don't pass the "allow", if there has an individual activity, we can continue the check, if not, just go to notAuthorized handle. 
I think there may be a better solution for my concern. Could anyone help to review the changes?